src/core/checkedint.d

```diff
@@ -567,7 +567,7 @@ unittest
 }
 
 /// ditto
-pragma(inline, true)
+pragma(inline, false) // false as workaround to http://issues.dlang.org/show_bug.cgi?id=20906
 long muls()(long x, long y, ref bool overflow)
```

Comment:
Why is this function a template?

Reply:
There was a reason, but I don't recall what it was at the moment, and it wasn't documented.

Reply:
[It was to make core.checkedint usable in betterC.](https://github.com/dlang/druntime/pull/2426)  After [#11236](https://github.com/dlang/dmd/pull/11236) is pulled, if DMD is taught how to inline functions like this, `pragma(inline, true)` would be a better solution than a dummy template.